GoMoney/Common/InputView/AccessoryView.swift

```diff
@@ -0,0 +1,106 @@
+/// https://github.dev/sag333ar/InputViews
+
+import UIKit
+
+public class AccessoryView: UIView {
+    lazy var toolBar = UIToolbar()
+
+    lazy var doneItem: UIBarButtonItem = {
+        let doneItem = UIBarButtonItem(
+            barButtonSystemItem: .done,
+            target: self,
+            action: #selector(doneButtonTapped)
+        )
+
+        doneItem.setTextAttributes(font: UIFont(name: K.Font.nova, size: 16))
+
+        return doneItem
+    }()
+
+    lazy var addItem: UIBarButtonItem = {
+        let addItem = UIBarButtonItem(
+            barButtonSystemItem: .add,
+            target: self,
+            action: #selector(addButtonTapped)
+        )
+        return addItem
+    }()
+
+    var title = ""
+    var doneTapped: (() -> Void)?
+    var addTapped: (() -> Void)?
+    var shouldShowAdd: (() -> Bool) = { false }
```

Comment:
if just use for simple case, set true/false => should use normal variable instead of closure